### PR TITLE
Closes #115: Refine arkouda helm charts

### LINKED ISSUE #115: refine arkouda-helm-charts
Minor changes are needed for the arkouda-helm-charts:

1. upgrade to v2023.09.06 arkouda-docker images
2. add Role and RoleBinding files to arkouda-udp-server
3. narrow scope of ClusterRole/ClusterRoleBinding to Role/RoleBinding
4. make arkouda-udp-server and arkouda-udp-locale users configurable via values.yaml

### PR DESCRIPTION
**Pull request by hokiegeek2**
Closes #115 

This PR includes the following:

1. streamlined arkouda-udp-server chart
2. streamlined prometheus-arkouda-exporter
3. updated README files

### CODE REVIEW

File: arkouda-helm-charts/arkouda-udp-server/templates/service-rolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: arkouda-service-crud
+subjects:
+- kind: User
+  name: arkouda
```
Comment (pierce314159) [inline arkouda-helm-charts/arkouda-udp-server/templates/service-rolebinding.yaml:7]: should the `name` be `{{ .Values.user }}` like in the other rolebinding yamls?
```suggestion
- kind: User
  name: {{ .Values.user }}
```

**Review by pierce314159 — COMMENTED** [review]
only one question, looks good otherwise!

File: arkouda-helm-charts/arkouda-udp-server/templates/service-rolebinding.yaml (hunk 1 above)
Comment (pierce314159) [reply arkouda-helm-charts/arkouda-udp-server/templates/service-rolebinding.yaml:7]: @hokiegeek2 should this name be updated to match the other yamls?

**Review by pierce314159 — APPROVED** [review]

**Review by jaketrookman — APPROVED** [review]

File: arkouda-helm-charts/arkouda-udp-server/templates/service-rolebinding.yaml (hunk 1 above)
Comment (hokiegeek2) [reply arkouda-helm-charts/arkouda-udp-server/templates/service-rolebinding.yaml:7]: @pierce314159 wow, good catch! checking

Comment (hokiegeek2) [reply arkouda-helm-charts/arkouda-udp-server/templates/service-rolebinding.yaml]: @pierce314159 yup, excellent catch, fixed!

**Pull request merged**